proto/proto-backplane-grpc/Dockerfile

```diff
@@ -1,5 +1,13 @@
+FROM deephaven/go:local-build as go-build
+
+RUN go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28
+RUN go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.2
```

Comment:
could also consider moving these to our images. 

worth doing now, or wait until we land the go client api patch, and then make me fix it?